charts/kyverno/templates/deployment.yaml

```diff
@@ -28,7 +28,16 @@ spec:
       securityContext: {{ tpl (toYaml .) $ | nindent 8 }}
       {{- end }}
       {{- with .Values.affinity }}
-      affinity: {{ tpl (toYaml .) $ | nindent 8 }}
+      affinity: 
```

Comment:
Addressing [this](https://github.com/kyverno/kyverno/pull/1975/files#r647734434) review - I couldn't find how to add pod anti affinity to helm charts so I'm not sure if what I've done is correct. @realshuting could you please point me to the correct way of doing this? Thanks.

Initially I saw how elestic search had implemented this their [charts](https://github.com/elastic/helm-charts/blob/master/elasticsearch/values.yaml#L157-L163) so I did the same but I'm confused now if the change is required in the deployment.yaml or values.yaml file.